tasks/ingress-nginx.yml

```diff
@@ -13,4 +13,4 @@
     dest: "{{ rke2_data_path }}/server/manifests/rke2-ingress-nginx-config.yaml"
     owner: root
     group: root
-    mode: 0664
+    mode: 0660
```

Comment:
Modifying file and directory permissions to differ from those in the upstream RKE2 installation is not within the scope of this Ansible Role.